dashboard/app/controllers/safe_browsing_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SafeBrowsingController < ApplicationController
+  include SafeBrowsing
```

Comment:
I don't think you need to `include SafeBrowsing` here; this effectively mixes the `SafeBrowsing` module into the `SafeBrowsingController` class, but you're only calling a static method in that module.  If you just remove this line, does it still work?